src/services/insights/helper/index.js

```diff
@@ -0,0 +1,11 @@
+const { userModel } = require('../../user/model.js');
+
+const findSchool = async (userId) => {
+	const user = await userModel.findById(userId).select('schoolId');
+	const { schoolId } = user;
+	return schoolId || '';
```

Comment:
here you translate a `undefined` into an _empty string_, is that really required?
you must check user is not null too

Reply:
Will remove the whole function as I found a lookupSchool hook we could use instead.